### PR TITLE
add bar component

### PR DESCRIPTION
**Pull request by skurnia001**

### CODE REVIEW

File: src/@next/Bar/BarStyle.ts [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+  display: flex;
+  flex-direction: column-reverse;
+  align-items: flex-end;
+`;
+
+export const StyledBarContainer = styled.div`
+  position: relative;
+  margin: 0 auto;
+
+  @media ${Device.mobileS} {
```
Comment (ninariccimarie) [inline src/@next/Bar/BarStyle.ts]: we currently only have 2 Breakpoints defined for new design system. you can checkout how `${Breakpoints.large}` is used in the other new components

File: src/@next/Bar/BarStyle.ts [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+  flex-direction: column;
+`;
+
+export const StyledBarActionWrapper = styled.div`
+  display: flex;
+  flex-direction: column-reverse;
+  align-items: flex-end;
+`;
+
+export const StyledBarContainer = styled.div`
```
Comment (ninariccimarie) [inline src/@next/Bar/BarStyle.ts:42]: please add:
```
width: 100%;
padding: (50px left and right for large screens and 24px left and right for smaller screens)
```

to stretch the content.

from this:
Screenshot 2023-03-08 at 2 05 35 PM

to this:
Screenshot 2023-03-08 at 2 13 42 PM

File: src/@next/Bar/BarStyle.ts [hunk 3]
```diff
@@ -0,0 +1,93 @@
+import styled from 'styled-components';
+import { Device, ScreenSize } from '../..';
+import { Neutral } from '../utilities/colors';
+import { space8 } from '../utilities/spacing';
+
+export const StyledBar = styled.div`
+  display: flex;
+  flex-direction: row;
+  padding: 20px 0;
```
Comment (ninariccimarie) [inline src/@next/Bar/BarStyle.ts:9]: I think this is also dynamic based on screen size. can @chiahou confirm the spacing?

**Review by ninariccimarie — CHANGES_REQUESTED** [review]

File: src/@next/Bar/BarStyle.ts (hunk 3 above)
Comment (chiahou) [reply src/@next/Bar/BarStyle.ts:9]: Actually its height should be fixed. I've highlighted some details [in the guideline](https://www.figma.com/file/87fhOdF8vk1Uiy20bCqS85/Sirius-Web---UI-Kit-3.0-%5BQ4-2022%5D?node-id=2795-37391&t=wXUtowNgaYzQL2Fj-4). There are 2 types of stickey bar, one used for a page and one for a section. Could you please help to align the text styles, paddings, heights and button sizes for desktop and mobile view?

File: src/@next/Bar/Bar.tsx [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+    tertiaryAction,
+    position,
+  }: BarProps,
+  ref
+) {
+  const headerMarkup = () => {
+    if (subheading) {
+      return (
+        <>
+          <Typography as="div" variant="body2">
```
Comment (chiahou) [inline src/@next/Bar/Bar.tsx]: text color for heading should be Neutral.B18 (#2d2d2d)

File: src/@next/Bar/Bar.tsx [hunk 5]
```diff
@@ -0,0 +1,78 @@
...
+            {heading}
+          </Typography>
+          <Typography as="div" variant="subtitle2" color={Colors.Neutral.B40}>
+            {subheading}
+          </Typography>
+        </>
+      );
+    }
+    return (
+      <Typography as="div" variant="headline6">
```
Comment (chiahou) [inline src/@next/Bar/Bar.tsx]: text color for heading should be Neutral.B18 (#2d2d2d)

File: src/@next/Bar/Bar.tsx [hunk 6]
```diff
@@ -0,0 +1,78 @@
...
+      </Typography>
+    );
+  };
+
+  return (
+    <StyledBar data-align={position} ref={ref}>
+      <StyledBarContainer>
+        <StyledBarHeaderWrapper>{headerMarkup()} </StyledBarHeaderWrapper>
+        <StyledBarActionWrapper>
+          <ButtonGroup>
```
Comment (chiahou) [inline src/@next/Bar/Bar.tsx:64]: button size should be "default" for mobile view (max-width: 768px)

Comment (skurnia001) [reply src/@next/Bar/Bar.tsx:64]: @ninariccimarie Is this automatically handled, just like Typography?

**Review by chiahou — APPROVED** [review]
qc passed, thanks

**Review by ninariccimarie — APPROVED** [review]
LGTM 🚀 thank you!

**Pull request merged**